mypyc/genops.py

```diff
@@ -2091,6 +2091,27 @@ def visit_decorator(self, o: Decorator) -> None:
         raise NotImplementedError
 
     def visit_del_stmt(self, o: DelStmt) -> None:
+        if isinstance(o.expr, IndexExpr):
+            self.visit_del_item(o.expr)
+        elif isinstance(o.expr, TupleExpr):
+            for expr_item in o.expr.items:
+                if isinstance(expr_item, IndexExpr):
+                    self.visit_del_item(expr_item)
```

Comment:
This ignores non-indexexprs if they are nested within a tuple, when it would be better to trip an assert. Maybe something recursive is what we want here.